Sources/NIOHTTP2/HTTP2FrameConvertible.swift

```diff
@@ -23,3 +23,39 @@ protocol HTTP2FramePayloadConvertible {
     /// Makes a `HTTP2Frame.FramePayload`.
     var payload: HTTP2Frame.FramePayload { get }
 }
+
+extension HTTP2FrameConvertible where Self: HTTP2FramePayloadConvertible {
```

Comment:
This extension seems to properly simply belong on `HTTP2FramePayloadConvertible`, no?

Reply:
From a purely implementation perspective yes. But we use only this property in the context of writing (i.e. we require `HTTP2FrameConvertible` rather than `HTTP2FramePayloadConvertible`). WDYT?

Reply:
Hmmm. I guess, yeah. 👍